eduNEXT/xblock-controlled-navigation
# fix: resize xblock iframe on content changes

### Description
Add a simplified script based on [this edx-platform script](https://github.com/openedx/edx-platform/blob/master/lms/templates/courseware/courseware-chromeless.html#L119-L217) in the xblock template to resize the iframe each time we navigate between child components. 

### How to Test
1. Install this branch.
2. Add `controlled_navigation` to **Studio** > **Course** > **Settings** > **Advanced Settings**.
3. Add a new component of **Controlled Navigation** in a unit.
4. Add children to the component with different heights.
5. In the LMS, move backward and forward between components using the **Next** and **Previous** buttons. You should see the total content of all components.
